Opcodes/py/pythonopcodes.h

```diff
@@ -21,7 +21,13 @@
 #ifndef _pycsound_pythonopcodes_h_
 #define _pycsound_pythonopcodes_h_
 
-#include <Python.h>
+#ifdef _DEBUG
+# undef _DEBUG
+#  include <Python.h>
+# define _DEBUG
+#else
+# include <Python.h>
+#endif
```

Comment:
No problem. It's required because in debug builds it looks for python*_d.lib which won't exist.